software/deluge.yml

```
@@ -0,0 +1,14 @@
+name: Deluge
+website_url: https://deluge-torrent.org/
+description: Deluge is a lightweight, Free Software, cross-platform BitTorrent client.
+licenses:
+  - GPL-3.0
+platforms:
+  - Python
+  - JavaScript
```

```suggestion
  - deb
```

> Values for platform are the main **server-side** requirements for the software

I also suggest adding `deb` since it is [available in Debian repositories](https://packages.debian.org/bookworm/deluge-web)